src/javascript/app/common/active_symbols.js

```diff
@@ -170,6 +171,24 @@ const ActiveSymbols = (() => {
         extend(trade_markets_list, getSubmarkets(active_symbols));
         return trade_markets_list;
     };
+    
+    // The unavailable underlyings are only offered on deriv.com.
+    const getAvailableUnderlyings = (markets_list) => {
+        const unavailable_underlyings = ['BOOM500', 'BOOM1000', 'CRASH500', 'CRASH1000', 'stpRNG'];
```

Comment:
I think we can have this variable defined outside of this function so we don't declare it every time, since the value is static